.github/workflows/build-mudlet.yml

```diff
@@ -137,6 +146,12 @@ jobs:
         restore-keys: ccache-${{matrix.os}}-${{matrix.compiler}}-${{matrix.qt}}-
       if: matrix.os != 'windows-latest'
 
+    - name: (Linux/macOS) Set build info
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+      run: |
+        ${{github.workspace}}/CI/travis.validate_deployment.sh
```

Comment:
Perhaps we should lose the "travis" from these names...?

Reply:
We can, I just left it in for ease of reviewing.